### PR TITLE
Adding GUI Mirroring - issue #890

### PR DESCRIPTION
**Pull request by Ashamandi**
This pull request will solve most of mirroring issues. 
others issues will be solving at another PR as I need to add some changes at .js files.

### CODE REVIEW

**Comment by Ashamandi** [comment]
Hello @ceejay,
Could you please review this changes ?

File: editor/sass/jquery.scss [hunk 1]
```diff
@@ -114,6 +128,12 @@
     }
 }
 
+[dir="rtl"] .ui-dialog-buttonset button {
+    &.leftButton {
```
Comment (dceejay) [inline editor/sass/jquery.scss]: is & correct ?

**Comment by dceejay** [comment]
css looks fine
two thoughts - 1) there are a lot of lines like   if ($("body").attr("dir") == "rtl") { 
Is there not (or could there be) an overall global already defined in the js somewhere that could be used rather than going through jquery every time ?
2) the fa-icons have the classes to rotate and flip already - see http://fontawesome.io/examples/
not sure if it would be easier to use them or not.

**Comment by Ashamandi** [comment]
About your thoughts: 
1- I agree with you, so please review my last commit
2- about fa-icons, using different class name to rotate fa-icons will require to append this class using js code, so I preferred to rotate it myself at scss files using the existing classes .... please let me know if you prefer the other approach.

**Comment by dceejay** [comment]
Hi,
1) - hmm - was more thinking there must already be some variable that this (larger bidi/rtl) work is already setting via some option... so will need integrating with the wider team and shouldn't need to add another one. (but maybe it's needed for stand-alone testing)
2) I have no preference - just making sure we hadn't missed something. Happy to leave as you have it.

**Pull request closed without merging**